src/PrestaShopBundle/DependencyInjection/PrestaShopExtension.php

```diff
@@ -26,16 +26,23 @@
 
 namespace PrestaShopBundle\DependencyInjection;
 
+use PrestaShop\PrestaShop\Adapter\Module\Repository\ModuleRepository;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
 /**
  * Adds main PrestaShop core services to the Symfony container.
  */
-class PrestaShopExtension extends Extension
+class PrestaShopExtension extends Extension implements PrependExtensionInterface
 {
+    public function __construct()
+    {
+        $this->activeModulesPaths = (new ModuleRepository(_PS_ROOT_DIR_, _PS_MODULE_DIR_))->getActiveModulesPaths();
```

Comment:
I think you can replace their values by using config
That would mean, create some kind of configuration for the bundle: 



```yaml
prestashop: 
    root_dir: !php/const _PS_ROOT_DIR
    module_dir: !php/const _PS_MODULE_DIR_
```

And then you can fetch values inside the container builder

Reply:
It looks like the bundle has no configuration and I'm not sure how to create one for it nor if I should declare the Extension as a service to pass it parameters